src/_pytest/mark/structures.py

```diff
@@ -377,18 +371,21 @@ def get_unpacked_marks(obj) -> List[Mark]:
 
 
 def normalize_mark_list(mark_list: Iterable[Union[Mark, MarkDecorator]]) -> List[Mark]:
-    """Normalize marker decorating helpers to mark objects.
+    """
+    Normalize an iterable of Mark or MarkDecorator objects into a list of marks
+    by retrieving the `mark` attribute on MarkDecorator instances.
 
-    :type List[Union[Mark, Markdecorator]] mark_list:
-    :rtype: List[Mark]
+    :param mark_list: Iterable[Union[Mark, MarkDecorator]] to normalize
```

Comment:
```suggestion
    :param mark_list: marks to normalize
```

However I think describing the parameters here is a bit redundant and could be removed. 👍